check_manually.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+wget https://raw.githubusercontent.com/offa/android-foss/master/README.md
```

Comment:
This shouldn't be necessary when called locally any more, but I can fix this afterwards.